qiskit_ibm_runtime/base_runtime_job.py

```diff
@@ -135,11 +135,23 @@ def cancel_result_streaming(self) -> None:
             return
         self._ws_client.disconnect(WebsocketClientCloseCode.CANCEL)
 
+    def usage(self) -> Dict[str, Any]:
+        """Return job usage in seconds."""
+        try:
+            metrics = self._api_client.job_metadata(self.job_id())
+            return metrics.get("usage")
```

Comment:
If we add a `usage()` method what exactly should be returned? This would return:

```
{'quantum_seconds': 0, 'seconds': 0}
```

but if quantum_seconds and seconds are the same, we can just returned one of them

Reply:
Yes we should return just 1